currency.js

```diff
@@ -0,0 +1,61 @@
+var https = require('https')
+
+function convertCurrency(amount, fromCurrency, toCurrency, cb) {
```

Comment:
can you add a comment above this declaration with the arguments to the callback? 
for example, i think it's `cb(error, value)` but it was hard to tell at first, i had to "figure it out" from the code